en/firefox/privacy/book.ftl

```diff
@@ -0,0 +1,208 @@
+
+privacy-book-its-also-possible = It’s also possible that nobody’s account got hijacked but your uncle still sends you a fraudulent link because he simply doesn’t know better. If you don’t know the site and prefer to be safe over being sorry, do a quick web search before clicking!
+
+privacy-book-look-at-the = Look at the bigger picture:
+
+privacy-book-we-already-talked = We already talked about how people tend to like convenience – remember? When it comes to information we consume it might be nice to just see our attitude and expectations confirmed because it’s pleasant and calming. But does that make the information accurate? Do we get the big picture or just a rather small extract?
+
+privacy-book-you-can-of = <strong>You can, of course, have your favorites</strong> but check out different outlets from time to time to learn about different opinions and give other perspectives a chance.
+
+privacy-book-ask-yourselves-do = <strong>Ask yourselves:</strong> Do the different outlets report on the issues you care about? And is the information shared more or less consistent or does it differ a lot? That’s the best way to battle filter bubbles and, ultimately, also filter bubbles and, ultimately, also misinformation.
+
+# This sentence will only be shown to "DE" locale because the blog post is in German only.
+privacy-book-de-blog-post = If you want to learn more about the latter, here’s <a href="https://blog.mozilla.org/firefox/how-to-spot-real-fake-news/">an interesting blog post</a> that can be helpful.
+
+privacy-book-disable-unnecessary-location = Disable (unnecessary) location services:
+
+privacy-book-todays-mobile-devices = Today’s mobile devices typically have the ability to determine, store and share their location. That’s convenient for navigation or when you’ve lost your phone. At the same time though, this quite sensitive piece of information may also be used by apps, even if you don’t want it.
+
+privacy-book-many-social-media = <strong>Decide case by case:</strong> Many social media apps offer to share your location when you post something. It’s up to you if you want others to know where you are and it’s totally fine to say no.
+
+privacy-book-for-each-of = <strong>Decide for each app:</strong> For each of the apps on your device that wants access to your location information, you can decide if you want to share it. All major operating systems allow for you to change your settings so that you limit access for individual or all apps.
+
+privacy-book-another-reason-to = <strong>Avoid background localization:</strong> Another reason to do so is that some apps don’t only save your location when you use them – many of them also run the location service in the background and may create complete motion profiles of you over time. The app vendor may then use this information for a lot of purposes, even sell it, which may not suit you.
```

Comment:
`... some apps don’t only save your location ...` change `don't only save` to `not only save`.